File: CarCareHub_/CarCareHub.Services/Database/Klijent.cs
using System;
using System.Collections.Generic;

namespace CarCareHub.Services.Database;

public partial class Klijent
{
    public int KlijentId { get; set; }

    public string? Ime { get; set; }

    public string? Prezime { get; set; }

    public string? Username { get; set; }

    public string? Email { get; set; }

    public string? Password { get; set; }

    public string? Spol { get; set; }

    public string? BrojTelefona { get; set; }

    public int? GradId { get; set; }

    public virtual ICollection<ChatKlijentServis> ChatKlijentServis { get; set; } = new List<ChatKlijentServis>();

    public virtual ICollection<ChatKlijentZaposlenik> ChatKlijentZaposleniks { get; set; } = new List<ChatKlijentZaposlenik>();

    public virtual Grad? Grad { get; set; }
}

File: CarCareHub_/UI_/flutter_mobile/lib/models/klijent.dart
import 'package:json_annotation/json_annotation.dart';
part 'klijent.g.dart';

@JsonSerializable()
class Klijent {
  int klijentId;
  String? ime;
  String? prezime;
  String? username;
  String? email;
  String? password;
  String? passwordAgain;
  String? lozinkaSalt;
  String? lozinkaHash;
  String? spol;
  String? brojTelefona;
  int? gradId;
  int? ulogaId;
String? adresa;
bool? vidljivo;


  Klijent(this.klijentId, this.adresa, this.vidljivo, this.ime,this.prezime,this.username,this.email,this.password,this.lozinkaSalt,this.lozinkaHash,this.spol,this.brojTelefona,this.gradId,this.passwordAgain);

  
  factory Klijent.fromJson(Map<String,dynamic> json) => _$KlijentFromJson(json);


  Map<String,dynamic> toJson() => _$KlijentToJson(this);
}